packages/polaris-viz/src/utilities/getAxisOptions.ts

```diff
@@ -11,6 +11,10 @@ export function getYAxisOptionsWithDefaults(
     integersOnly: false,
     fixedWidth: false,
     maxYOverride: null,
+    label: '',
```

Comment:
Could we move these off the generic `getAxisOptions` so that they're specific to the `Grid`?